MAINTAINERS.md

```diff
@@ -132,67 +130,27 @@ During your review, consider the following points:
     the author is the one most familiar with the changes they are introducing, they are the party
     most likely to resolve conflicts correctly.
 
-
 ## Workflow for merging Pull Requests
 
-Check which branch the PR was made against — patches made against the `develop` branch should only
-be merged to `develop`. Patches made against the `master` branch may be merged to only `develop`, or
-both `develop` AND `master`. The criteria to use in this latter situation is: does the patch
-introduce anything new to the API? does it potentially mark a change in behavior of any sort? If the
-answer to either of these is "yes", merge only to the `develop` branch.
-
-To sum up:
-
-- If the commit is a bugfix, merge to **both** `master` **and** `develop`.
-- If the commit is a new feature, merge to `develop` **only**.
-
-### Start the process with a clean checkout
-
-Before you begin merging, do the following:
-
-```console
-$ git fetch origin
-$ git checkout master && git rebase origin/master
-$ git checkout develop && git rebase origin/develop
-```
-
-This will set each branch to the latest commit from the canonical repository.
-
-### Create a local merge branch
-
-Next, create a new branch locally for the change, based off the appropriate branch. Use the prefix
-`hotfix/` for fixes, and `feature/` for features; typically, use the PR number as the branch name.
-
-```console
-$ git checkout -b hotfix/2854 master
-$ git checkout -b feature/2719 develop
-```
+Pull Requests should only be merged into the correct release branches:
 
-### Pull in the changeset
+- If the PR is a bugfix, it should target one of the previous release branches.
+  The only exception is when the package is not yet released and therefore has only 
+  one release branch.
+- If the PR is a new feature, merge it to the latest release branch **only**.
 
-Now you're staged and ready to pull the changeset in. On the bar that occurs before comments, on the
-left side is a little "information" icon. Click on this, and you get a popup dialog showing the
-steps to take to manually merge. If you click the icon next to the second step, it will copy a "git
-pull" command to your clipboard. Paste this into your terminal.
+You may merge it by using the green "Merge" button in github. If you do so, use only the
+"Create a merge commit" option. Do **not** use "Squash and merge" or "Rebase and merge". 
 
-> #### Hub
->
-> GitHub develops a [hub](https://github.com/github/hub) command that provides a superset of
-> features for your git executable. It can make the following far simpler:
->
-> ```console
-> $ git merge <url to PR>
-> ```
->
-> We recommend using the hub command to make your life as a maintainer easier!
+Alternatively you may clone the repository and merge the PR locally. If you decide to do so, 
+do not apply additional changes other than changelog entries to the merge.
```

Comment:
This doesn't detail _how_ to merge. Should it be a standard merge, with a merge commit? Or a `merge --ff`? Or should we rebase and then `merge --ff`? (This discussion is related to my comments about using the button for merging. We need consistency, but also a good rationale for what we choose. And it needs to be the same whether using the button, or doing it at the CLI.)

Reply:
That's right much like above I actually don't know how to start here. At first do we want a permissive approach or restrictive approach. Both has its pros and cons. Permissive offers more flexibility, restrictive is more streamlined. Whatever the descition is: Here it should be clear to a Maintainer, what's good to go and what should not be done in either a permissive ("Don't do x, y, z - otherwise do as you think it suits best") or restrictive ("Only do x, y, z") way.